erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -795,3 +796,36 @@ def update_item_quantity(source, target, source_parent):
 	doc.set_item_locations()
 
 	return doc
+
+@frappe.whitelist()
+def start_job_cards(name):
+	job_cards = []
+	open_job_cards = frappe.db.get_all('Job Card', {
+		'work_order':name, 'status':('in', ['Open', 'Work In Progress', 'Material Transferred'])
+		}, ['*'])
+	for job_card in open_job_cards:
+		job = frappe.get_doc('Job Card', job_card.name)
```

Comment:
Use a separate function to be able to process individual job cards, we can re-use that later then. And call that function from both `start_job_cards` and `stop_job_cards`